### DISCUSSION TITLE
ILBMViewer - Deluxe Paint Image Viewer Online (.LBM and .ILBM) - Retro Reversing (Reverse Engineering)

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#197] [Post Comments]
# ILBMViewer - Deluxe Paint Image Viewer Online (.LBM and .ILBM) - Retro Reversing (Reverse Engineering)

RetroReversing, the home of retro game reverse engineering. Get resources, tutorials, and tools for analyzing video games and join the community today!

https://www.retroreversing.com/ilbm

**Comment by salivity** [comment]
never heard of this format before

**Comment by EminemMeetsGoku23** [comment]
This works on very retro Sierra games that are opened with Game Extractor. Now I just have to figure out the .snd files, they are heavily modified on an Educational Sierra game.
